README.md

```diff
@@ -9,24 +9,59 @@ Using Django and raw js to begin with, server side will be provided by individua
 
 ### Django template tag
 1. Include the `field_update` app in your django settings INSTALLED_APPS
-4. Ensure you have csrf_token cookie present by using `ensure_csrf_cookie` decorator on your view
-2. Identify the submit view and url you wish to send updates to
-3. load the field updater template tags library in the template you wish to place the updater 
+2. Ensure you have csrf_token cookie present by using `ensure_csrf_cookie` decorator on your view
+3. Identify the submit view and url you wish to send updates to
 
-`{% load field_updater_tags %}`
+if your api returns a Location header the componenet will subequently use that as the submit url
+
+4. load the field updater template tags library in the template you wish to place the updater 
 
-4. Use the field updater template tag passing the url to request to, and the attribute name and initial value as a key value pair
+`{% load field_updater_tags %}`
 
-{% url 'submit_url_name' some_arg=some_value as submit_url %}
+5. Use the field updater template tag passing the url to request to, and the attribute name and initial value as a key value pair
 
+`{% url 'submit_url_name' some_arg=some_value as submit_url %}`
 `{% field_updater submit_url=submit_url attribute_name=attribute_value %}`
-`{% field_updater submit_url=submit_url attribute_name=attribute_value allow_delete=True %}`
 
-The initial value will be displayed, on clicking the display an update text input will show, edit the value and click the tick, and a POST request will be submitted to `submit_url with formencoded data of `attribute_name=attribute_value`
+6. optionally pass instance specific options such as ifMatch or ifUnmodifiedSince to help prevent lost update issues
+
+The component will start sending ifMatch if your enpoint returns an ETag header, unless you pass ifMatch=False
+
+The component will start sending IfUnmodifiedSince headers if you endpoint returns a Last-Modified header, unless you pass IfUnmodifiedSince=False
+
+`{% field_updater submit_url=submit_url attribute_name=attribute_value ifMatch='an_etag' %}`
```

Comment:
Please have consistent naming conventions for all the arguments and avoid mixing camel and snake cases, as it's a bit confusing. Example: Based on other arg names (`submit_url`, `attribute_name`, etc), I would expect `ifMatch` to be `if_match`.

Reply:
fixed, documentation error!